tests/functional/test_truststore.py

```diff
@@ -56,6 +42,5 @@ def test_trustore_can_install(
     pip: PipRunner,
     package: str,
 ) -> None:
-    script.pip("install", "truststore")
     result = pip("install", package)
```

Comment:
Doesn't this require opt-in to be testing truststore?

Reply:
It is opt-in (we have the `--use-feature=truststore` flag in the `pip(...)` call), we no longer need to install `truststore` beforehand though.

Reply:
Ah thanks. I didn't spot the difference between `pip()` and `script.pip()`.